### DISCUSSION TITLE
is there a way to downgrade the dreambooth extention to an older version?

### DISCUSSION THREAD

**Discussion by eyeweaver** [d8ahazard/sd_dreambooth_extension#529] [Q&A]
Hello everybody

Today I've updated my dreambooth extension.. and I found that the parameters and options have changed.. and there's a new thing called "LORA".. now its really too slow compared to the older version.. and the results are not as good as they used to be.. I have no idea why.. and Idea how can I downgrade to an older release?

**Comment by Thomas-MMJ** [comment]
you can checkout a prior version using git,

go into the folder sd_dreambooth_extension  

then do

`git switch -c newBranch $(git rev-list -n1 --before=yyyy-mm-dd main)
`

https://stackoverflow.com/questions/6990484/how-to-checkout-in-git-by-date

**Comment by eyeweaver** [comment]
Thank you so much for your help..
I got this error message:
error: unknown switch `n'
any idea why?

**Reply by Thomas-MMJ** [reply]
Did you replace the yyyy-mm-dd with a date such as 2022-12-01 ?

No idea what causes the error.
